approved_tags.yaml

```diff
@@ -62,6 +62,7 @@
 - { tag: "reti", description: "Reti di comunicazione" }
 - { tag: "reverse-engineering", description: "Ingegneria inversa" }
 - { tag: "rilascio", description: "Rilascio di software in produzione o al pubblico" }
+- { tag: "ricerca", description: "Campi dell'informatica ancora in fase sperimentale" }
```

Comment:
un po' troppo specifico. Secondo me dovrebbe includere anche ricerca in campi maturi.

Reply:
Dopo la discussione nei commenti al thread pensavo ci si volesse riferire a quello che ho scritto. Ammetto però che così sia molto specifico, per cui ho corretto.